frontend/src/components/App/Notifications/index.tsx

```diff
@@ -133,43 +161,56 @@ export default function Notifications() {
   const { t } = useTranslation();
 
   useEffect(() => {
+    let notificationsToShow: Notification[] = [];
+    let currentNotifications = notifications;
+    let changed = false;
+
+    if (currentNotifications.length === 0) {
+      currentNotifications = JSON.parse(localStorage.getItem('notifications') || '[]');
+      changed = true;
```

Comment:
This block here causes the App.test.jsx to loop forever. Doesn't loop in the app though it seems.

Reply:
Happens when there are no events. I have updated the PR with a fix and the tests passed now, so I will merge it. Thanks.